### DISCUSSION TITLE
WiFi usage with Seeed Studio XIAO ESP32S3 (Sense) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#213] [Q&A]
# WiFi usage with Seeed Studio XIAO ESP32S3 (Sense) | Seeed Studio Wiki

WiFi usage with Seeed Studio XIAO ESP32S3.

https://wiki.seeedstudio.com/zh-CN/xiao_esp32s3_wifi_usage/

**Comment by Mario-Adlys** [comment]
hi, is the antenna mandatory on the Xiao ESP32s3? Or can it be used without it (don't you need a resistor if no antenna)?  Thanks.

Mario

**Reply by MatthewJeffson** [reply]
I think it is mandatory, if you want to get a good signal while you using the Wi-Fi function. :D
